common/src/main/scala/auditor/package.scala

```diff
@@ -1,26 +1,63 @@
 import java.net.URL
 
-import models.Topic
+import models.{Topic, TopicTypes}
+import play.api.Logger
 import play.api.libs.json.Json
+import play.api.libs.ws.WSClient
 
 import scala.concurrent.{ExecutionContext, Future}
+import scala.util.Try
 
 package object auditor {
```

Comment:
Yeah I think that's a good idea, starting by putting general package names (my bad :/)